content/documentation/pages/7-resources/3-faq.md

```diff
@@ -417,6 +417,40 @@ See the [compatibility matrix](https://github.com/spring-cloud/spring-cloud-stre
 
 <!--END_QUESTION-->
 
+<!--QUESTION#lifecycle-->
+
+Can I manage binding lifecycles?
+
+By default, bindings are started automatically when the application is initialized.
+Bindings implement the Spring `SmartLifecycle` interface.
+`SmartLifecycle` allows beans to be started in phases.
+Producer bindings are started in an early phase (`Integer.MIN_VALUE + 1000`).
+Consumer bindings are started in a late phase (`Integer.MAX_VALUE - 1000`).
+This leaves room in the spectrum such that user beans implementing `SmartLifecycle` can be started before producer bindings, after consumer bindings, or anywhere in between.
+
```

Comment:
If agree with above, then this whole sentence is no longer relevant.

Reply:
I think this is more succinct.